src/wizards/tokenSwapDealWizard/stages/tokenDetailsStage/tokenDetailsStage.ts

```diff
@@ -101,7 +102,7 @@ export class TokenDetailsStage {
 
       name: "",
       symbol: "",
-      decimals: 18,
+      decimals: TokenService.DefaultDecimals,
```

Comment:
use default value

Reply:
There are 2 PRs, that do the same change.
Reason: both PRs needed to change this to fix underlying issue

![image](https://user-images.githubusercontent.com/30693990/159568952-d9d5bc1a-eae8-4394-9b1d-13c7808bfabf.png)